packages/oc-template-handlebars-compiler/lib/compileServer.js

```diff
@@ -24,8 +24,13 @@ module.exports = (options, callback) => {
   compiler(config, (err, compiledServer) => {
     if (err) {
       return callback(err);
-    } else {
-      fs.ensureDirSync(publishPath);
+    }
+
+    fs.ensureDir(publishPath, err => {
```

Comment:
I moved many of these Sync functions to async. This should allow us to parallelise better and speed up overall compilation instead of being blocked by IO operations 👍